packages/fxa-payments-server/src/routes/Product/SubscriptionUpgrade/PlanUpgradeDetails.tsx

```diff
@@ -50,24 +121,76 @@ export const PlanUpgradeDetails = ({
 
       <PlanDetailsCard className="to-plan" plan={selectedPlan} />
 
-      <div className="py-6 border-t-0 plan-details-item font-semibold">
-        <Localized id="sub-update-total-label">
-          <div className="total-label">New total</div>
-        </Localized>
-
-        <Localized
-          id={`plan-price-${selectedPlan.interval}`}
-          vars={{
-            amount: getLocalizedCurrency(
-              selectedPlan.amount,
-              selectedPlan.currency
-            ),
-            intervalCount: selectedPlan.interval_count,
-          }}
-        >
-          <div className="total-price">{totalPrice}</div>
-        </Localized>
-      </div>
+      {loading ? (
+        <LoadingSpinner />
+      ) : (
+        <div className="py-6 border-t-0">
+          {!!subTotal && !!taxAmount && (
+            <>
+              <div className="plan-details-item">
+                <Localized id="plan-details-list-price">
+                  <div>List Price</div>
+                </Localized>
+
+                <Localized
+                  id={`list-price`}
+                  attrs={{ title: true }}
+                  vars={{
+                    amount: getLocalizedCurrency(subTotal, currency),
+                    intervalCount: interval_count,
+                  }}
+                >
+                  <div>{getLocalizedCurrencyString(subTotal, currency)}</div>
+                </Localized>
+              </div>
+
+              <div className="plan-details-item">
+                <Localized id="plan-details-tax">
+                  <div>Taxes and Fees</div>
+                </Localized>
+
+                <Localized
+                  id={`tax`}
```

Comment:
Q: Why ``id={`tax`}`` vs `id="tax"`?
and I guess same Q for L136 above, ``<Localized id={`list-price`}``; versus `<Localized id="sub-update-total-label">` below on L168.

Reply:
I'm assuming this was just a placeholder. It's been updated.